src/components/Quotes.jsx

```diff
@@ -26,8 +26,11 @@ export default function Quotes() {
   }, []);
 
   return (
-    <div>
-      <h1>{loading}</h1>
+    <div
+      style={{
+        padding: '20px',
+      }}>
+      <h2>{loading}</h2>
```

Comment:
- [ ] The "Math Magicians" text-based logo should appear on all the pages just like in the wireframe. The aim is for the general layout to match the given wireframe's. I suggest you set this up in your navigation so that you don't repeat code in your components 👍 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/235907962-c43d7557-81ea-45fa-b959-0b0288537b01.png)